lib/galaxy/workflow/modules.py

```diff
@@ -197,7 +197,7 @@ def get_runtime_inputs(self, **kwds):
         """
         return {}
 
-    def compute_runtime_state(self, trans, step_updates=None):
+    def compute_runtime_state(self, trans, step=None, step_updates=None):
```

Comment:
Can you document the new `step` parameter, please?

Reply:
Done below.